src/vm-memory/src/mmap.rs

```diff
@@ -0,0 +1,1191 @@
+// Copyright (C) 2019 Alibaba Cloud Computing. All rights reserved.
+//
+// Portions Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
nit: For this local file where we add dirty tracking support, these two might be better reversed.